contracts/Treasury.sol

```diff
@@ -59,18 +64,19 @@ contract OlympusTreasury is OlympusAccessControlled, ITreasury {
 
     /* ========== STATE VARIABLES ========== */
 
-    IOHM public immutable OHM;
-    IsOHM public sOHM;
+    IFLOOR public immutable FLOOR;
+    IsFLOOR public sFLOOR;
 
     mapping(STATUS => address[]) public registry;
     mapping(STATUS => mapping(address => bool)) public permissions;
-    mapping(address => address) public bondCalculator;
+    mapping(address => address) public override bondCalculator;
+    mapping(address => uint256) public _riskOffValuation; // 18 decimal in ETH terms 
```

Comment:
9 decimal in FLOOR terms i.e. PUNK would be `20_000_000_000_000` (20,000 FLOOR or 20 ETH)